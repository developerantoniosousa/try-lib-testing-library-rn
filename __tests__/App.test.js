import React from 'react';
import App from '../src/App';

import { render } from '@testing-library/react-native';

it('should render App without crash', () => {
  const tree = render(<App />);
  expect(tree).toMatchSnapshot();
});
