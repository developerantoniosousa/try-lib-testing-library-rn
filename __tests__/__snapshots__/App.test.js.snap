// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App without crash 1`] = `
Object {
  "asJSON": [Function],
  "baseElement": <View
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      collapsable={true}
      pointerEvents="box-none"
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <SafeAreaView>
        <ScrollView
          contentInsetAdjustmentBehavior="automatic"
          style={
            Object {
              "backgroundColor": "#F3F3F3",
            }
          }
        >
          <View
            accessibilityIgnoresInvertColors={true}
            style={
              Object {
                "backgroundColor": "#F3F3F3",
                "paddingBottom": 40,
                "paddingHorizontal": 32,
                "paddingTop": 96,
              }
            }
          >
            <Image
              accessibilityRole="image"
              source={
                Object {
                  "testUri": "../Libraries/NewAppScreen/components/logo.png",
                }
              }
              style={
                Array [
                  Object {
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  },
                  Object {
                    "height": undefined,
                    "width": undefined,
                  },
                  Object {
                    "marginBottom": -192,
                    "marginLeft": -128,
                    "opacity": 0.2,
                    "overflow": "visible",
                    "resizeMode": "cover",
                  },
                ]
              }
            />
            <Text
              style={
                Object {
                  "color": "#000",
                  "fontSize": 40,
                  "fontWeight": "600",
                  "textAlign": "center",
                }
              }
            >
              Welcome to React
            </Text>
          </View>
          <View
            style={
              Object {
                "backgroundColor": "#FFF",
              }
            }
          >
            <View
              style={
                Object {
                  "marginTop": 32,
                  "paddingHorizontal": 24,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#000",
                    "fontSize": 24,
                    "fontWeight": "600",
                  }
                }
              >
                Step One
              </Text>
              <Text
                style={
                  Object {
                    "color": "#444",
                    "fontSize": 18,
                    "fontWeight": "400",
                    "marginTop": 8,
                  }
                }
              >
                Edit 
                <Text
                  style={
                    Object {
                      "fontWeight": "700",
                    }
                  }
                >
                  App.js
                </Text>
                 to change this screen and then come back to see your edits.
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 32,
                  "paddingHorizontal": 24,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#000",
                    "fontSize": 24,
                    "fontWeight": "600",
                  }
                }
              >
                See Your Changes
              </Text>
              <Text
                style={
                  Object {
                    "color": "#444",
                    "fontSize": 18,
                    "fontWeight": "400",
                    "marginTop": 8,
                  }
                }
              >
                <Text>
                  Press 
                  <Text
                    style={
                      Object {
                        "fontWeight": "700",
                      }
                    }
                  >
                    Cmd + R
                  </Text>
                   in the simulator to reload your app's code.
                </Text>
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 32,
                  "paddingHorizontal": 24,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#000",
                    "fontSize": 24,
                    "fontWeight": "600",
                  }
                }
              >
                Debug
              </Text>
              <Text
                style={
                  Object {
                    "color": "#444",
                    "fontSize": 18,
                    "fontWeight": "400",
                    "marginTop": 8,
                  }
                }
              >
                <Text>
                  Press 
                  <Text
                    style={
                      Object {
                        "fontWeight": "700",
                      }
                    }
                  >
                    Cmd + D
                  </Text>
                   in the simulator or
                   
                  <Text
                    style={
                      Object {
                        "fontWeight": "700",
                      }
                    }
                  >
                    Shake
                  </Text>
                   your device to open the React Native debug menu.
                </Text>
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 32,
                  "paddingHorizontal": 24,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#000",
                    "fontSize": 24,
                    "fontWeight": "600",
                  }
                }
              >
                Aprenda Mais
              </Text>
              <Text
                style={
                  Object {
                    "color": "#444",
                    "fontSize": 18,
                    "fontWeight": "400",
                    "marginTop": 8,
                  }
                }
              >
                Read the docs to discover what to do next:
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 32,
                  "paddingHorizontal": 24,
                }
              }
            >
              <View
                style={
                  Object {
                    "backgroundColor": "#DAE1E7",
                    "height": 1,
                  }
                }
              />
              <
                accessibilityRole="button"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "flexWrap": "wrap",
                    "justifyContent": "space-between",
                    "paddingVertical": 8,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1292B4",
                      "flex": 2,
                      "fontSize": 18,
                      "fontWeight": "400",
                    }
                  }
                >
                  The Basics
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#444",
                      "flex": 3,
                      "fontSize": 18,
                      "fontWeight": "400",
                      "paddingVertical": 16,
                    }
                  }
                >
                  Explains a Hello World for React Native.
                </Text>
              </>
              <View
                style={
                  Object {
                    "backgroundColor": "#DAE1E7",
                    "height": 1,
                  }
                }
              />
              <
                accessibilityRole="button"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "flexWrap": "wrap",
                    "justifyContent": "space-between",
                    "paddingVertical": 8,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1292B4",
                      "flex": 2,
                      "fontSize": 18,
                      "fontWeight": "400",
                    }
                  }
                >
                  Style
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#444",
                      "flex": 3,
                      "fontSize": 18,
                      "fontWeight": "400",
                      "paddingVertical": 16,
                    }
                  }
                >
                  Covers how to use the prop named style which controls the visuals.
                </Text>
              </>
              <View
                style={
                  Object {
                    "backgroundColor": "#DAE1E7",
                    "height": 1,
                  }
                }
              />
              <
                accessibilityRole="button"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "flexWrap": "wrap",
                    "justifyContent": "space-between",
                    "paddingVertical": 8,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1292B4",
                      "flex": 2,
                      "fontSize": 18,
                      "fontWeight": "400",
                    }
                  }
                >
                  Layout
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#444",
                      "flex": 3,
                      "fontSize": 18,
                      "fontWeight": "400",
                      "paddingVertical": 16,
                    }
                  }
                >
                  React Native uses flexbox for layout, learn how it works.
                </Text>
              </>
              <View
                style={
                  Object {
                    "backgroundColor": "#DAE1E7",
                    "height": 1,
                  }
                }
              />
              <
                accessibilityRole="button"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "flexWrap": "wrap",
                    "justifyContent": "space-between",
                    "paddingVertical": 8,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1292B4",
                      "flex": 2,
                      "fontSize": 18,
                      "fontWeight": "400",
                    }
                  }
                >
                  Components
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#444",
                      "flex": 3,
                      "fontSize": 18,
                      "fontWeight": "400",
                      "paddingVertical": 16,
                    }
                  }
                >
                  The full list of components and APIs inside React Native.
                </Text>
              </>
              <View
                style={
                  Object {
                    "backgroundColor": "#DAE1E7",
                    "height": 1,
                  }
                }
              />
              <
                accessibilityRole="button"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "flexWrap": "wrap",
                    "justifyContent": "space-between",
                    "paddingVertical": 8,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1292B4",
                      "flex": 2,
                      "fontSize": 18,
                      "fontWeight": "400",
                    }
                  }
                >
                  Navigation
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#444",
                      "flex": 3,
                      "fontSize": 18,
                      "fontWeight": "400",
                      "paddingVertical": 16,
                    }
                  }
                >
                  How to handle moving between screens inside your application.
                </Text>
              </>
              <View
                style={
                  Object {
                    "backgroundColor": "#DAE1E7",
                    "height": 1,
                  }
                }
              />
              <
                accessibilityRole="button"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "flexWrap": "wrap",
                    "justifyContent": "space-between",
                    "paddingVertical": 8,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1292B4",
                      "flex": 2,
                      "fontSize": 18,
                      "fontWeight": "400",
                    }
                  }
                >
                  Networking
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#444",
                      "flex": 3,
                      "fontSize": 18,
                      "fontWeight": "400",
                      "paddingVertical": 16,
                    }
                  }
                >
                  How to use the Fetch API in React Native.
                </Text>
              </>
              <View
                style={
                  Object {
                    "backgroundColor": "#DAE1E7",
                    "height": 1,
                  }
                }
              />
              <
                accessibilityRole="button"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "flexWrap": "wrap",
                    "justifyContent": "space-between",
                    "paddingVertical": 8,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1292B4",
                      "flex": 2,
                      "fontSize": 18,
                      "fontWeight": "400",
                    }
                  }
                >
                  Help
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#444",
                      "flex": 3,
                      "fontSize": 18,
                      "fontWeight": "400",
                      "paddingVertical": 16,
                    }
                  }
                >
                  Need more help? There are many other React Native developers who may have the answer.
                </Text>
              </>
              <View
                style={
                  Object {
                    "backgroundColor": "#DAE1E7",
                    "height": 1,
                  }
                }
              />
              <
                accessibilityRole="button"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "flexWrap": "wrap",
                    "justifyContent": "space-between",
                    "paddingVertical": 8,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1292B4",
                      "flex": 2,
                      "fontSize": 18,
                      "fontWeight": "400",
                    }
                  }
                >
                  Follow us on Twitter
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#444",
                      "flex": 3,
                      "fontSize": 18,
                      "fontWeight": "400",
                      "paddingVertical": 16,
                    }
                  }
                >
                  Stay in touch with the community, join in on Q&As and more by following React Native on Twitter.
                </Text>
              </>
            </View>
          </View>
        </ScrollView>
      </SafeAreaView>
    </View>
  </View>,
  "container": <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <SafeAreaView>
      <ScrollView
        contentInsetAdjustmentBehavior="automatic"
        style={
          Object {
            "backgroundColor": "#F3F3F3",
          }
        }
      >
        <View
          accessibilityIgnoresInvertColors={true}
          style={
            Object {
              "backgroundColor": "#F3F3F3",
              "paddingBottom": 40,
              "paddingHorizontal": 32,
              "paddingTop": 96,
            }
          }
        >
          <Image
            accessibilityRole="image"
            source={
              Object {
                "testUri": "../Libraries/NewAppScreen/components/logo.png",
              }
            }
            style={
              Array [
                Object {
                  "bottom": 0,
                  "left": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                },
                Object {
                  "height": undefined,
                  "width": undefined,
                },
                Object {
                  "marginBottom": -192,
                  "marginLeft": -128,
                  "opacity": 0.2,
                  "overflow": "visible",
                  "resizeMode": "cover",
                },
              ]
            }
          />
          <Text
            style={
              Object {
                "color": "#000",
                "fontSize": 40,
                "fontWeight": "600",
                "textAlign": "center",
              }
            }
          >
            Welcome to React
          </Text>
        </View>
        <View
          style={
            Object {
              "backgroundColor": "#FFF",
            }
          }
        >
          <View
            style={
              Object {
                "marginTop": 32,
                "paddingHorizontal": 24,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#000",
                  "fontSize": 24,
                  "fontWeight": "600",
                }
              }
            >
              Step One
            </Text>
            <Text
              style={
                Object {
                  "color": "#444",
                  "fontSize": 18,
                  "fontWeight": "400",
                  "marginTop": 8,
                }
              }
            >
              Edit 
              <Text
                style={
                  Object {
                    "fontWeight": "700",
                  }
                }
              >
                App.js
              </Text>
               to change this screen and then come back to see your edits.
            </Text>
          </View>
          <View
            style={
              Object {
                "marginTop": 32,
                "paddingHorizontal": 24,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#000",
                  "fontSize": 24,
                  "fontWeight": "600",
                }
              }
            >
              See Your Changes
            </Text>
            <Text
              style={
                Object {
                  "color": "#444",
                  "fontSize": 18,
                  "fontWeight": "400",
                  "marginTop": 8,
                }
              }
            >
              <Text>
                Press 
                <Text
                  style={
                    Object {
                      "fontWeight": "700",
                    }
                  }
                >
                  Cmd + R
                </Text>
                 in the simulator to reload your app's code.
              </Text>
            </Text>
          </View>
          <View
            style={
              Object {
                "marginTop": 32,
                "paddingHorizontal": 24,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#000",
                  "fontSize": 24,
                  "fontWeight": "600",
                }
              }
            >
              Debug
            </Text>
            <Text
              style={
                Object {
                  "color": "#444",
                  "fontSize": 18,
                  "fontWeight": "400",
                  "marginTop": 8,
                }
              }
            >
              <Text>
                Press 
                <Text
                  style={
                    Object {
                      "fontWeight": "700",
                    }
                  }
                >
                  Cmd + D
                </Text>
                 in the simulator or
                 
                <Text
                  style={
                    Object {
                      "fontWeight": "700",
                    }
                  }
                >
                  Shake
                </Text>
                 your device to open the React Native debug menu.
              </Text>
            </Text>
          </View>
          <View
            style={
              Object {
                "marginTop": 32,
                "paddingHorizontal": 24,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#000",
                  "fontSize": 24,
                  "fontWeight": "600",
                }
              }
            >
              Aprenda Mais
            </Text>
            <Text
              style={
                Object {
                  "color": "#444",
                  "fontSize": 18,
                  "fontWeight": "400",
                  "marginTop": 8,
                }
              }
            >
              Read the docs to discover what to do next:
            </Text>
          </View>
          <View
            style={
              Object {
                "marginTop": 32,
                "paddingHorizontal": 24,
              }
            }
          >
            <View
              style={
                Object {
                  "backgroundColor": "#DAE1E7",
                  "height": 1,
                }
              }
            />
            <
              accessibilityRole="button"
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "flexWrap": "wrap",
                  "justifyContent": "space-between",
                  "paddingVertical": 8,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#1292B4",
                    "flex": 2,
                    "fontSize": 18,
                    "fontWeight": "400",
                  }
                }
              >
                The Basics
              </Text>
              <Text
                style={
                  Object {
                    "color": "#444",
                    "flex": 3,
                    "fontSize": 18,
                    "fontWeight": "400",
                    "paddingVertical": 16,
                  }
                }
              >
                Explains a Hello World for React Native.
              </Text>
            </>
            <View
              style={
                Object {
                  "backgroundColor": "#DAE1E7",
                  "height": 1,
                }
              }
            />
            <
              accessibilityRole="button"
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "flexWrap": "wrap",
                  "justifyContent": "space-between",
                  "paddingVertical": 8,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#1292B4",
                    "flex": 2,
                    "fontSize": 18,
                    "fontWeight": "400",
                  }
                }
              >
                Style
              </Text>
              <Text
                style={
                  Object {
                    "color": "#444",
                    "flex": 3,
                    "fontSize": 18,
                    "fontWeight": "400",
                    "paddingVertical": 16,
                  }
                }
              >
                Covers how to use the prop named style which controls the visuals.
              </Text>
            </>
            <View
              style={
                Object {
                  "backgroundColor": "#DAE1E7",
                  "height": 1,
                }
              }
            />
            <
              accessibilityRole="button"
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "flexWrap": "wrap",
                  "justifyContent": "space-between",
                  "paddingVertical": 8,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#1292B4",
                    "flex": 2,
                    "fontSize": 18,
                    "fontWeight": "400",
                  }
                }
              >
                Layout
              </Text>
              <Text
                style={
                  Object {
                    "color": "#444",
                    "flex": 3,
                    "fontSize": 18,
                    "fontWeight": "400",
                    "paddingVertical": 16,
                  }
                }
              >
                React Native uses flexbox for layout, learn how it works.
              </Text>
            </>
            <View
              style={
                Object {
                  "backgroundColor": "#DAE1E7",
                  "height": 1,
                }
              }
            />
            <
              accessibilityRole="button"
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "flexWrap": "wrap",
                  "justifyContent": "space-between",
                  "paddingVertical": 8,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#1292B4",
                    "flex": 2,
                    "fontSize": 18,
                    "fontWeight": "400",
                  }
                }
              >
                Components
              </Text>
              <Text
                style={
                  Object {
                    "color": "#444",
                    "flex": 3,
                    "fontSize": 18,
                    "fontWeight": "400",
                    "paddingVertical": 16,
                  }
                }
              >
                The full list of components and APIs inside React Native.
              </Text>
            </>
            <View
              style={
                Object {
                  "backgroundColor": "#DAE1E7",
                  "height": 1,
                }
              }
            />
            <
              accessibilityRole="button"
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "flexWrap": "wrap",
                  "justifyContent": "space-between",
                  "paddingVertical": 8,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#1292B4",
                    "flex": 2,
                    "fontSize": 18,
                    "fontWeight": "400",
                  }
                }
              >
                Navigation
              </Text>
              <Text
                style={
                  Object {
                    "color": "#444",
                    "flex": 3,
                    "fontSize": 18,
                    "fontWeight": "400",
                    "paddingVertical": 16,
                  }
                }
              >
                How to handle moving between screens inside your application.
              </Text>
            </>
            <View
              style={
                Object {
                  "backgroundColor": "#DAE1E7",
                  "height": 1,
                }
              }
            />
            <
              accessibilityRole="button"
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "flexWrap": "wrap",
                  "justifyContent": "space-between",
                  "paddingVertical": 8,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#1292B4",
                    "flex": 2,
                    "fontSize": 18,
                    "fontWeight": "400",
                  }
                }
              >
                Networking
              </Text>
              <Text
                style={
                  Object {
                    "color": "#444",
                    "flex": 3,
                    "fontSize": 18,
                    "fontWeight": "400",
                    "paddingVertical": 16,
                  }
                }
              >
                How to use the Fetch API in React Native.
              </Text>
            </>
            <View
              style={
                Object {
                  "backgroundColor": "#DAE1E7",
                  "height": 1,
                }
              }
            />
            <
              accessibilityRole="button"
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "flexWrap": "wrap",
                  "justifyContent": "space-between",
                  "paddingVertical": 8,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#1292B4",
                    "flex": 2,
                    "fontSize": 18,
                    "fontWeight": "400",
                  }
                }
              >
                Help
              </Text>
              <Text
                style={
                  Object {
                    "color": "#444",
                    "flex": 3,
                    "fontSize": 18,
                    "fontWeight": "400",
                    "paddingVertical": 16,
                  }
                }
              >
                Need more help? There are many other React Native developers who may have the answer.
              </Text>
            </>
            <View
              style={
                Object {
                  "backgroundColor": "#DAE1E7",
                  "height": 1,
                }
              }
            />
            <
              accessibilityRole="button"
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "flexWrap": "wrap",
                  "justifyContent": "space-between",
                  "paddingVertical": 8,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#1292B4",
                    "flex": 2,
                    "fontSize": 18,
                    "fontWeight": "400",
                  }
                }
              >
                Follow us on Twitter
              </Text>
              <Text
                style={
                  Object {
                    "color": "#444",
                    "flex": 3,
                    "fontSize": 18,
                    "fontWeight": "400",
                    "paddingVertical": 16,
                  }
                }
              >
                Stay in touch with the community, join in on Q&As and more by following React Native on Twitter.
              </Text>
            </>
          </View>
        </View>
      </ScrollView>
    </SafeAreaView>
  </View>,
  "debug": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByHintText": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByDisplayValue": [Function],
  "findByHintText": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByHintText": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByDisplayValue": [Function],
  "getByHintText": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByHintText": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByDisplayValue": [Function],
  "queryByHintText": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
